source/installation-guide/wazuh-agent/wazuh-agent-package-aix.rst

```diff
@@ -14,6 +14,8 @@ The deployment of a Wazuh agent on an AIX system uses deployment variables that
 
 #. To start the deployment process, download the `AIX installer <https://packages.wazuh.com/|WAZUH_CURRENT_MAJOR_AIX|/aix/wazuh-agent-|WAZUH_CURRENT_AIX|-|WAZUH_REVISION_AIX|.aix.ppc.rpm>`_. 
 
+.. note:: You need root user privileges to run all the commands described below.
+
```

Comment:
Can you add this to the AIX deployment too?

Reply:
I don't get this. What are you referring to?

Reply:
In AIX deployment the root note is not specified either

Reply:
I'm sorry, I got confused with the file path when doing the revision